recipe/meta.yaml

```diff
@@ -1,14 +1,14 @@
 # tensorflow-estimator meta data file
 {% set name = "tensorflow-estimator" %}
-{% set version = "2.3.0" %}
+{% set version = "2.4" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   git_url: https://github.com/tensorflow/estimator.git
-  git_rev: v{{ version }}
+  git_rev: r{{ version }}
  
 build:
   number: 1
```

Comment:
Build number update

Reply:
Since we are moving up to a new package version we want to keep the build number at 1.